src/services/doppler-legacy-client.doubles.ts

```diff
@@ -106,4 +107,17 @@ export class HardcodedDopplerLegacyClient implements DopplerLegacyClient {
     await timeout(1500);
     return { success: true };
   }
+
+  public async getBannerData(lang: string, type: string, page: string) {
```

Comment:
Why are you adding `getBannerData` into legacy-client? This information comes from a different source than doppler, comes from `sites` right? Perhaps it could be set aside in a different service. What do you think?

Reply:
maybe, but sites is from doppler too dont? makes sense?

Reply:
_DopplerSitesClient_?